srcpkgs/revive/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'revive'
 pkgname=revive
-version=1.1.4
+version=1.2.1
 revision=1
 build_style=go
-go_ldflags="-X main.version=$version -X main.builtBy=xbps"
+go_ldflags="-X github.com/mgechev/revive/cli.version=$version"
```

Comment:
```suggestion
go_ldflags="-X cli.version=$version"
```
would this work?

Reply:
Yes, mgechev/revive@d2ab719a0ee3513500eb36460b2d49a90bbbd6c1.
I also removed `builtBy` because it's unused unless built time is also set <https://github.com/mgechev/revive/blob/31fbdb18332df7d1db76f8223e5dfb0285cd46aa/cli/main.go#L151>.

Reply:
Sorry, I misread your comment. I don't know if the suggestion would work, I have to test.

Reply:
It doesn't work, it prints `version (devel)`.